consumption.tex

```diff
@@ -0,0 +1,50 @@
+\section{Consumption}
+
+We describe how the target blockchain can verify a burn which happened on the source blockchain. Specifically, we will study the case of Bitcoin and Ethereum as source blockchains. Most cryptocurrencies on the market are adaptations of either of these. We define a party which we call a \emph{prover} who is responsible for constructing a proof of the event and sending it to the target blockchain. The proof we need to send is one of transaction inclusion in the honestly adopted chain of the source blockchain. We will denote this proof as a triple $\beta = \sf(txRaw, txInclusion, blockHeader, blockConnection)$.
```

Comment:
triplet

Reply:
Smoother intro to the section. Perhaps say "We *now* describe..."

Reply:
Have we already spoken about source/target blockchains at this point?

Reply:
Avoid saying "we will" using a future tense. Just say "we study".

Reply:
The sentence "Most cryptocurrencies on the market are adaptions of either of these" is speculative. If you want to make it less speculative, we need to support this claim and quantify it, which I think is not the purpose of this paper. We can just remove that sentence.

Reply:
There isn't necessary *the* honestly adopted chain; there can be multiple honestly adopted chains. Rephrase.

Reply:
We aren't defining "transaction inclusion" here.

Reply:
Should we say at this point what are txRaw, txInclusion, blockHeader and blockConnection briefly?

Reply:
Are we sure we want to make this section explicitly about Bitcoin and Ethereum? Perhaps we can generalize somehow and make it abstract.